signer/src/transaction_coordinator.rs

```diff
@@ -365,30 +368,38 @@ where
 
     /// Construct and coordinate WSTS signing rounds for sBTC transactions on Bitcoin,
     /// fulfilling pending deposit and withdraw requests.
-    #[tracing::instrument(skip_all, fields(bitcoin_chain_tip))]
+    #[tracing::instrument(skip_all)]
```

Comment:
I guess you're doing this in several places now as the chain tip is included in a higher-up span?

Reply:
Yes, they'll be in the logs already since we add it earlier in the call stack.